<launch>
  <arg name="input/twist" default="/localization/twist" />

  <node name="motion_evaluator" exec="motion_evaluator" pkg="planning_evaluator" output="screen">
    <remap from="~/input/twist" to="$(var input/twist)" />
  </node>
</launch>
